misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/ChronoZonedDateTimeAsStringAssertionSpec.kt

```diff
@@ -0,0 +1,212 @@
+
+    include(object : ChronoZonedDateTimeAssertionSpec(
+        fun1(::isBefore),
+        fun1(::isBeforeOrEqual),
+        fun1(::isAfter),
+        fun1(::isAfterOrEqual),
+        fun1(::isEqual),
+        describePrefix
+    ) {})
+
+    val isBeforeFun = isBefore.lambda
+    val isBeforeOrEqualFun = isBeforeOrEqual.lambda
+    val isAfterFun = isAfter.lambda
+    val isAfterOrEqualFun = isAfterOrEqual.lambda
+    val isEqualFun = isEqual.lambda
+
+    val subject = ZonedDateTime.now() as ChronoZonedDateTime<*>
+    val now = expect(subject)
+
+    describe("wrong format") {
+        listOf(
+            "20200102030405666Z",
+            "20200102030405Z",
+            "202001020304Z",
+            "20200102Z",
+            "2020-1-02T03:04Z",
+            "2020-01-2T03:04Z",
+            "20-01-02T03:04Z",
+            "2020-01-02T3:04Z",
+            "2020-01-02T03:4Z",
+            "2020-01-02T03:04:5Z",
+            "2020-01-02t03:04Z",
+            "2020-01-02T03:04Z+5",
+            "2020-01-02T03:04Z+5:0",
+            "2020-01-02T03:04Z+05:6",
+            "2020-01-02T03:04Z-5",
+            "2020-01-02T03:04Z-5:06",
+            "2020-01-02T03:04Z-05:6"
+        ).forEach { value ->
+            context(value) {
+                it("${isBefore.name} throws a DateTimeParseException") {
+                    expect {
+                        now.isBeforeFun(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("${isBeforeOrEqual.name} throws a DateTimeParseException") {
+                    expect {
+                        now.isBeforeOrEqualFun(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("${isAfter.name} throws a DateTimeParseException") {
+                    expect {
+                        now.isAfterFun(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("isAfterOrEqual throws a DateTimeParseException") {
+                    expect {
+                        now.isAfterOrEqualFun(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+                it("${isEqual.name} throws a DateTimeParseException") {
+                    expect {
+                        now.isEqualFun(value)
+                    }.toThrow<DateTimeParseException> { messageContains("could not be parsed") }
+                }
+            }
+        }
+    }
+
+    describe("allowed shortcuts") {
+        mapOf(
+            "2020-01-02T03:04:05Z" to ZonedDateTime.of(2020, 1, 2, 3, 4, 5, 0, ZoneOffset.UTC) as ChronoZonedDateTime<*>,
+            "2020-01-02T03:04Z" to ZonedDateTime.of(2020, 1, 2, 3, 4, 0, 0, ZoneOffset.UTC),
+            "2020-01-02Z" to ZonedDateTime.of(2020, 1, 2, 0, 0, 0, 0, ZoneOffset.UTC)
```

Comment:
I have not thought about this shortcut but I actually like it :+1: 
Can you also add "2020-01-02+01:30" which should result in `2020-01-02T00:00:00+01:30`

Reply:
Ok, I dropped the named zones. 
I think I implemented all your suggestions. 
Tell me what you think of it now 😄